pheps/phep-0001.md

```diff
@@ -0,0 +1,338 @@
+
+PHEP editors do not make judgments on the content or merits of PHEPs. They merely do the administrative & editorial part. They may participate in the community discussion of a PHEP but have no obligation to be advocates.
+
+Disagreements with editors must be mediated by PyHC leadership (as documented in [PHEP audience](#phep-audience)) and they may impose sanctions, including removal of the editor role.
+
+# PHEP 1 Acceptance
+<a name="phep-1-acceptance"></a>
+This PHEP is subject to the same review and acceptance process that it proposes.
+
+# Rationale
+<a name="rationale"></a>
+PHEP 1 is based on [PEP 1](https://peps.python.org/pep-0001/), which is a process that has credibility in the Python community and has proven appropriate for a fairly large group, the PyHC community being (by definition) larger than the community for any one PyHC package. From this basis, changes were made to make the process relevant to PyHC and to streamline where practicable.
+
+# Rejected Ideas
+<a name="rejected-ideas"></a>
+
+## ASCII encoding
+<a name="ascii-encoding"></a>
+As originally proposed PHEPs were limited to ASCII where practicable. This was deemed overly restrictive; authors should have some freedom of format and style, and it's up to the authors and editors to ensure this is both reasonable and consistent within a PHEP. CommonMark specifies Unicode without an encoding.
+
+## Recording names at acceptance
+<a name="recording-names-at-acceptance"></a>
+There was consideration of recording the names of people present when a PHEP is accepted, or who specifically want to "sign on". This was decided to be out-of-scope for the PHEP process; the notes of the meeting should capture relevant information.
+
+## Source formatting
+<a name="source-formatting"></a>
+Certain ideas regarding the format of the Markdown source were removed as more complicated than their benefit: making the source the version of record over the rendered, notes on specifics of rendering, and trying to get pretty diffs of Markdown.
+
+# Open Issues
+<a name="open-issues"></a>
+## Immutability of PHEPs
+<a name="immutability-of-pheps"></a>
+PHEPs are proposed to be [immutable](#phep-maintenance) once Final, requiring replacement rather than modification. This is not universally accepted.
+
+### In favor
+Immutability maintains clarity of reference, "PHEP-x" rather than "PHEP-x version y", and avoids confusion arising from rolling updates. It encourages getting the PHEP correct before first acceptance. Finally, it removes the need to define and document an update process.
+
+A PHEP can be updated by introducing a new PHEP that starts with and builds on the original text. Everything is then open for discussion. This encourages complete re-evaluation if the community reaches the point where something is important enough to revisit, and discourages fixing a typo that doesn't hurt anything.
```

Comment:
Need to emphasize the role of the editors in keeping minor but embarrassing issues from slipping through! (That's part of the getting it right the first time process).

Reply:
[New wording](https://github.com/jtniehof/standards/blob/e335aacd1346f0876f5a54d9d617bf3c11ec2e32/pheps/phep-0001.md?plain=1#L283) on the editor's role here.